structures/message.js

```diff
@@ -0,0 +1,21 @@
+const s = require('superstruct')
+const Uuid = s.define('Uuid', require('is-uuid').v4)
```

Comment:
```suggestion
const Uuid = s.define('Uuid', isUUid.v4)
```

Do not put require in another location that the top of the code `const isUuid = require('is-uuid');`